app/javascript/packages/document-capture/components/address-search.tsx

```diff
@@ -63,9 +80,15 @@ function AddressSearch({ onAddressFound = () => {}, registerField }: AddressSear
           hint={t('in_person_proofing.body.location.po_search.address_search_hint')}
         />
       </ValidatedField>
-      <Button type="submit" className="margin-y-5" onClick={handleAddressSearch}>
+      <SpinnerButton
+        isWide
```

Comment:
Not directly related to the changes, but bost of our other buttons are "big", and I'd assume we'd want the same here as well.

```suggestion
        isWide
        isBig
```